### PR TITLE
Allow to set 'max' for brightness

### PR DESCRIPTION
**Pull request by micw**
Hello,
this PR add support for "max" to the light slider.

My use case is a bit uncommon:  I'm using light.mqtt to control some timers (e.g. garden water) where brightness is actually used for minutes. With this change (and aligning "max" with "brightness scale"), it works pretty well. Another use case could be lights that shall not be dimed to >X percent (e.g. for temperature reasons).

### CODE REVIEW

**Comment by micw** [comment]
BTW, I'd also like to create a PR that allows to set a state_template. But I could not find an easy way to use templates for this.

File: src/light-controller.js [hunk 1]
```diff
@@ -53,6 +53,8 @@ export class LightController extends Controller {
   }
   get _max() {
     switch (this.attribute) {
+      case "brightness":
+        return this._config.max || 100
```
Comment (thomasloven) [inline src/light-controller.js]: This is not necessary. `this.max` will only get `this._max` if `this._config.max` is not set. See controller.js.

Comment (micw) [reply src/light-controller.js]: Right, I changed it

**Comment by micw** [comment]
Would you be open for another PR that makes the brightness unit configurable?
https://github.com/thomasloven/lovelace-slider-entity-row/blob/master/src/light-controller.js#L126

**Comment by thomasloven** [comment]
Sorry for letting this lay for so long.
I do not like the idea of this since it would give the user a false sense of security.
While the slider would be blocked from setting the brightness too high, other mechanisms in Home Assistant wouldn't have a second thought before burning the light to a crisp.

It's better to implement this security limit elsewhere.

**Comment by micw** [comment]
I have just upgraded to the latest release. With the absolute "brightness" which already honors the max scale settings of the entity, this issue is solved pretty well. Closing it now.

**Pull request closed without merging**